components/policy-mgt/org.wso2.carbon.policy.mgt.core/src/main/java/org/wso2/carbon/policy/mgt/core/dao/impl/PolicyDAOImpl.java

```diff
@@ -1343,4 +1343,32 @@ private int readHighestPriorityOfPolicies() throws PolicyManagerDAOException {
         return priority;
     }
 
+    @Override
+    public int getPolicyCount() throws PolicyManagerDAOException {
+
+        Connection conn;
+        PreparedStatement stmt = null;
```

Comment:
Yes, we do. The 'stmt' variable should be initialized prior to passing to 'cleanupResources(stmt, resultset)' method.